motr/m0crate/crate_index.c

```diff
@@ -247,12 +253,6 @@ static void cr_time_measure_end(struct cr_time_measure_ctx *t)
 	t->elapsed = cr_time_measure_elapsed_now(t);
```

Comment:
Seems setting t->ts_next is missing here, please add

Reply:
Fixed.